docs/DeveloperGuide.md

```diff
@@ -881,12 +881,106 @@ testers are expected to do more *exploratory* testing.
 
    1. Download the jar file and copy into an empty folder
 
-   1. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
+   1. Run jar file in command prompt: Expected: Shows the GUI with a set of sample data. The window size may not be optimum.
```

Comment:
Can change to `Run jar file in command prompt and enter 'java -jar wanderlust.jar'`.